UnityProject/Assets/Scripts/Items/Others/Horn.cs

```diff
@@ -0,0 +1,118 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.Networking;
+
+/// <summary>
+///     Indicates an object that emits sound upon activation (bike horn/air horn...)
+/// </summary>
+public class Horn : Interactable<HandActivate, PositionalHandApply>
+{
+	public string Sound;
+	public float Cooldown = 0.2f;
+
+	//todo: emit HONK particles
+
+	/// <summary>
+	/// Chance of causing minor ear injury when honking next to a living being
+	/// </summary>
+	[Range(0f, 1f)]
+	public float CritChance = 0.1f;
+
+	[Range( 0f, 100f )]
+	public float CritDamage = 5f;
+
+	private bool allowUse = true;
+	private float randomPitch => Random.Range( 0.7f, 1.2f );
+
+	private IEnumerator CoolDown()
+	{
+		allowUse = false;
+		yield return WaitFor.Seconds(Cooldown);
+		allowUse = true;
+	}
```

Comment:
the cooldown usage is a bit weird, honking with my hand will make me unable to use it on another player until the cd ends, even if WillInteract() is accepted
names like CoolDown() and Cooldown are also a bit confusing

Reply:
are you planning to honk faster than 5 HPS (honks per second)? don’t know of any clowns that can honk this fast
it’s more of a precaution against script abuse

Reply:
oh, I get what you mean. added cooldown check to WillInteract instead